src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_params.py

```diff
@@ -32,9 +32,11 @@
 register_cli_argument('batch account', 'account_name', batch_name_type, options_list=('--name', '-n'))
 register_cli_argument('batch account create', 'location', location_type, help='The region in which to create the account.')
 register_cli_argument('batch account create', 'tags', tags_type, help="Space separated tags in 'key[=value]' format.")
-register_extra_cli_argument('batch account create', 'storage_account', help='The storage account name or resource ID to be used for auto storage.', validator=storage_account_id)
```

Comment:
@xingwu1 - I couldn't see a reason for these to be registered as 'extra' arguments as opposed to just arguments, so a changed them, please let me know if I need to change it back!